styleguide-app/Examples/SegmentedControl.elm

```diff
@@ -70,6 +70,32 @@ example =
     }
 
 
+coloredIcons : List Svg
+coloredIcons =
+    [ UiIcon.flag
+    , UiIcon.sprout
+    , UiIcon.star
+    , UiIcon.sapling
+    , Svg.withColor Colors.greenDark UiIcon.attention
```

Comment:
this one overflows the bounds of the container, even in your screenshot. Should that be possible?

Reply:
It does indeed! it's an issue with the segmented control itself -- see:
 
<img width="661" alt="Screen Shot 2020-07-29 at 1 15 01 PM" src="https://user-images.githubusercontent.com/8811312/88848581-846c9280-d19d-11ea-8d93-307d30db338b.png">

I'm not sure if it matters to us yet, but I think there's some value in showing limitations within the styleguide too. Easily fixable on the use-side by setting a height on the icon too, I think..

Reply:
The ❗ is just really excited!